tex/homology/cup-product.tex

```diff
@@ -577,11 +623,258 @@ \section{Wedge sums}
 
 Thus $S^2 \vee S^4$ and $\CP^2$ are not homotopy equivalent.
 
+Intuitively, what the proof above says is:
+\begin{moral}
+	The nontrivial $4$-cocycle $a_4 \in H^4(S^2 \vee S^4; \ZZ)$
+	has nothing to do with the $2$-cocycle $a_2$,
+	while the $4$-cocycle $\alpha^2 \in H^4(\CP^2)$ is the cup product $\alpha
+	\smile \alpha$ of the $2$-cocycle $\alpha$ with itself.
+\end{moral}
+
+The exercise below would be much easier to visualize, apart from the fact that $\RP^2$ is
+nonorientable --- in fact, we have already seen
+above why $\alpha \smile \alpha \neq 0$ for the nonzero element $\alpha \in H^1(\RP^2)$.
+
+\begin{exercise}
+	Similarly, show that $S^1 \vee S^2$ and $\RP^2$ are not homotopy equivalent by showing
+	$\wt H^\bullet(S^1 \vee S^2; \Zc 2) \not\cong \wt H^\bullet(\RP^2; \Zc 2)$,
+	even though each graded component is isomorphic.
+\end{exercise}
+
+\section{Cross product}
+
+In this section, we will define the cross product.
+
+\subsection{Motivation}
+
+Roughly speaking, the motivation is the following:
+\begin{moral}
+	If $X$ has a $m$-dimensional hole and $Y$ has a $n$-dimensional hole, then $X \times Y$ has a
+	$(m + n$)-dimensional hole.
+\end{moral}
```

Comment:
I don't know if my intuition is fully correct yet... hopefully it will get refined at the end of this semester. (doesn't help that I don't really have a reference for the whole thing. Especially the cross product of differential form, which I can't find anywhere)